packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -9,6 +9,13 @@ it('should render a default radio field', () => {
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render a default radio field without crashing if it has an undefined child', () => {
+  const undefinedChild = undefined;
+  const radioField = (<RadioField legend="Default RadioField">{ undefinedChild }</RadioField>);
```

Comment:
Should this example just be
```<RadioField legend="Default RadioField" />```
or are we actually wanting to prevent  passing undefined as the children value?

Reply:
Prevent from passing undefined as the children value. In other words, we are testing the RadioField component with a child component which is undefined.